telnyx/api_resources/abstract/nested_resource_class_methods.py

```diff
@@ -4,17 +4,27 @@
 from telnyx.six.moves.urllib.parse import quote_plus
 
 
-def nested_resource_class_methods(resource, path=None, operations=None):
+def nested_resource_class_methods(
+    resource, path=None, operations=None, quote_params=True
```

Comment:
The parameter is called `quote_params` but I don't see how you're passing URL parameters. Parameters as in the function, not any argument? How about `quote_path_parts`?

Is this a workaround for call control not using URL-safe IDs?

Reply:
Thanks for the comment @nicktimko .
I agree that this parameter name is kinda confuse. `quote_path_parts` would be much more clear.
And yes, this is actually for both. 
For call-control we haven't URL safe and for `path` has been always escaped 
e.g `actions/extend` becomes `actions%2Fextend`

Reply:
@nicktimko @IgnasiBosch I sent another patch based on this comment. Thanks again for that, very valuable.
My solution: We're using quote_plus that is pretty much a proxy method to quote() (with some logic), but overriding  `safe` func param.
see: [cpyton/urllib/parse.py](https://github.com/python/cpython/blob/eed5e9a9562d4dcd137e9f0fc7157bc3373c98cc/Lib/urllib/parse.py#L787)

Now, we're just using builtin quote_plus safe param to ensure paths will always be escaped. Also, we have a "list" of safe chars we want to use in the path `id` param.  (such =,- )